exim/service/running.sls

```diff
@@ -13,5 +13,6 @@ exim/service/running:
   service.running:
     - name: {{ exim.service }}
     - enable: true
+    - reload: true
```

Comment:
(Probably) Won't be required, see below.

Reply:
`reload` signals that the site is capable of reloading instead of restarting. In this case, is preferred over restarting.